src/System.Management.Automation/engine/Subsystem/DscSubsystem/ICrossPlatformDsc.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.ObjectModel;
+using System.Collections.Generic;
+using System.Management.Automation.Internal;
+using System.Management.Automation.Language;
+
+namespace System.Management.Automation.Subsystem
+{
+    /// <summary>
+    /// Interface for implementing a cross platform desired state configuration component.
+    /// </summary>
+    public interface ICrossPlatformDsc : ISubsystem
+    {
+        /// <summary>
+        /// Subsystem kind.
+        /// </summary>
+        SubsystemKind ISubsystem.Kind => SubsystemKind.CrossPlatformDsc;
+
+        /// <summary>
+        /// Dsc initializer function.
+        /// </summary>
+        void LoadDefaultCimKeywords(Collection<Exception> errors);
```

Comment:
Can we rename this method to not use CIM in it? Since we are moving away from CIM, it's better to not use this term in the new DSC implementation.

Reply:
Method renamed.